python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -365,22 +365,24 @@ def __get__(self, obj, cls):
             return cls._schema[name]
 
         if self._getter_name:
-            return self._bind_method(obj, self._getter_name)()
+            returnable = self._bind_method(obj, self._getter_name)()
+            if isinstance(returnable, GangaObject):
+                returnable._setParent(self)
+            return returnable
```

Comment:
I think this is a specific bug that some piece of code is inserting a proxy object into Core where it shouldn't be for the obj to be proxied here.
If this is related to GangaLHCb then it's likely a LHCb bug which I'd say we fix in 6.1.16.